platforms/core-configuration/kotlin-dsl-tooling-builders/src/crossVersionTest/groovy/org/gradle/kotlin/dsl/tooling/builders/r83/KotlinBuildScriptModelCrossVersionSpec.groovy

```diff
@@ -28,6 +28,7 @@ class KotlinBuildScriptModelCrossVersionSpec extends AbstractKotlinScriptModelCr
     @Issue("https://github.com/gradle/gradle/issues/25555")
     def "single project with parallel build should not emit configuration resolution deprecation warning"() {
         given:
+        requireIsolatedUserHome()
```

Comment:
💅 please add a comment explaining why as that is faster to discover than the commit message / PR description.

Also please amend the commit message to include the explanation in it